### ISSUE TITLE
bug(express-engine): 

### ISSUE THREAD

**Issue by CaerusKaru** [angular/universal#806]
* **I'm submitting a ...**
```
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support request here, see note at the top of this template.
```

* **What modules are related to this Issue?**
```
- [ ] aspnetcore-engine
- [x] express-engine
- [ ] hapi-engine
```

* **Do you want to request a *feature* or report a *bug*?**

Bug report

* **What is the current behavior?**

Latest version of express-engine is not exported properly, and thus cannot be loaded by express server

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** by creating a github repo.

1. Create new Universal capable app
2. Create an express server to serve the app
3. Try to use express-engine to render the Universal app
4. Receive error

* **What is the expected behavior?**

Express-engine should be exported and available for use in latest version

* **What is the motivation / use case for changing the behavior?**

Express-engine cannot be used if it cannot be found by the express app

* **Please tell us about your environment:**

- Angular version: 5.0.0-beta.7
- Language: [all | TypeScript 2.5.2 | ES6/7 | ES5 ]
- OS:  [all | Mac OS X | Windows | **Linux** ]
- Platform: [all | **NodeJs** | Java | PHP | .NETCore | Ruby]

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

The error log:

```
Error: Cannot find module '@nguniversal/express-engine'
```

**Comment by CaerusKaru** [comment]
To clarify, this is only an issue with 5.0.0-beta.0, not 1.0.0-beta.3

**Comment by Toxicable** [comment]
Can you make a minimal reproduction of this? I don't get this error in my testing.
There is an error with the current version and v5 but that's been resolved by https://github.com/angular/universal/commit/5c705cb51362bd448c0b8ac293da867a3a1913f4 however, what you are describing is something else.
I've attempted to reproduce it by doing:
* `git clone https://github.com/angular/universal-starter`
* `cd universal-starter`
* update to `5.0.0-beta.7`
* `npm run serve:dynamic`

**Comment by CaerusKaru** [comment]
Exact reproduction following your steps:

- git clone https://github.com/angular/universal-starter
- cd universal-starter
- update to angular 5.0.0-beta.7
- update to express-engine 5.0.0-beta.0
- npm run serve:dynamic

**Comment by CaerusKaru** [comment]
Closed to bundle with #805
